cirq-core/cirq/protocols/apply_mixture_protocol.py

```diff
@@ -24,7 +24,6 @@
 from cirq._doc import doc_private
 from cirq.protocols import qid_shape_protocol
 from cirq.protocols.apply_unitary_protocol import apply_unitary, ApplyUnitaryArgs
-from cirq.protocols.mixture_protocol import mixture
```

Comment:
Can you please adjust the apply_mixture docstring at line 186 below?

Reply:
The implementation of that aspect of `apply_mixture` hasn't changed. Interestingly it's always been capable of handling gates/ops. The only change in `apply_mixture` is that previously it called `mixture`, which always returned the "mixture of unitaries" format. The new code calls `_mixture_` instead, which can return gates, and which can be run more efficiently via `_apply_unitary_` if the gate implements it.

Basically this docstring sounds like the original plan was to implement it the way this PR has implemented it, but then it got forgotten about.